relayer/relays/beacon/header/syncer/syncer.go

```diff
@@ -394,15 +394,17 @@ func (s *Syncer) GetHeaderUpdate(blockRoot common.Hash, checkpoint *cache.Proof)
 		return scale.HeaderUpdatePayload{}, fmt.Errorf("beacon header to scale: %w", err)
 	}
 
-	executionPayloadScale, err := api.CapellaExecutionPayloadToScale(block.GetExecutionPayload(), s.activeSpec)
+	executionHeaderBranch, err := s.getExecutionHeaderBranch(block)
 	if err != nil {
 		return scale.HeaderUpdatePayload{}, err
 	}
 
-	executionHeaderBranch, err := s.getExecutionHeaderBranch(block)
+	// Todo: Upgrade to Deneb based on fork epoch
+	executionPayloadScale, err := api.CapellaExecutionPayloadToScale(block.GetExecutionPayload(), s.activeSpec)
 	if err != nil {
 		return scale.HeaderUpdatePayload{}, err
 	}
+	versionedExecutionPayloadHeader := scale.VersionedExecutionPayloadHeader{Capella: &executionPayloadScale}
```

Comment:
@vgeddes as you mentioned in this PR we just introduced `VersionedExecutionPayloadHeader` and made `Capella` the only item for now and left `Deneb` support to future PRs.